tests/integration/dynamic_ip_test.py

```diff
@@ -382,7 +381,9 @@ def test_ipv4_dhcp_off_and_option_on(dhcpcli_up):
 
     libnmstate.apply(desired_state)
 
-    dhcp_cli_current_state = statelib.show_only((DHCP_CLI_NIC,))[INTERFACES][0]
+    dhcp_cli_current_state = statelib.show_only((DHCP_CLI_NIC,))[
+        Interface.KEY
+    ][0]
```

Comment:
The result of black formatting this looks terrible. Could you please use a temporary variable to make it more aesthetic?
`current_state = statelib.show_only((DHCP_CLI_NIC,))`